src/js/ReaderSettingsDialog.js

```diff
@@ -1,3 +1,4 @@
+var foo=5;
```

Comment:
Ready for more code review.

Thanks.


On 7/19/2016 10:29 AM, Daniel Weck wrote:
>
> In src/js/ReaderSettingsDialog.js 
> <https://github.com/readium/readium-js-viewer/pull/540#discussion_r71374286>:
>
> > @@ -1,3 +1,4 @@
> > +var foo=5;
>
> Can the |foo| var be removed? (it looks unused)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/readium/readium-js-viewer/pull/540/files/41229f6bade6f63f04851ed68233d9b5a76478b3..642bf5d9a886bdb2e765d61e05ee366db8b5a0f4#r71374286>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFGivWjZRlpqwGZfVm43fz9eiwqrPOsZks5qXPuBgaJpZM4I8jm9>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194